app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 4,
+        } %>
 
-            <ul class="govuk-list govuk-list--bullet">
-              <li>health</li>
-              <li>education</li>
-              <li>culture</li>
-              <li>the environment</li>
-              <li>transport</li>
-            </ul>
-          </div>
-        </div>
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Local government",
-              heading_level: 3,
-              margin_bottom: 5,
-            } %>
+        <p class="govuk-body">Putting data in people’s hands can help them have more of a say in the reform of public services. On <%= link_to "data.gov.uk", "https://data.gov.uk/", class: "govuk-link", rel: "external" %> you can easily find, review and use information about our country and communities - for example, to develop web applications.</p>
+      </section>
 
-            <p class="govuk-body">Councils make and carry out decisions on local services. Many parts of England have 2 tiers of local government: county councils and district, borough or city councils.</p>
+      <section class="govuk-!-margin-bottom-9">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Devolved government",
+          id: "devolved-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
 
-            <p class="govuk-body">In some parts of the country, there’s just one tier of local government providing all the functions, known as a ‘unitary authority’. This can be a city, borough or county council – or it may just be called ‘council’. As well as these, many areas also have parish or town councils.</p>
+        <p class="govuk-body">In <%= link_to "Scotland", "https://www.gov.scot/", class: "govuk-link", rel: "external" %>, <%= link_to "Wales", "https://www.gov.wales/", class: "govuk-link", rel: "external" %> and <%= link_to "Northern Ireland", "https://www.northernireland.gov.uk/", class: "govuk-link", rel: "external" %>, devolved administrations are responsible for many domestic policy issues, and their Parliaments/Assemblies have law-making powers for those areas.</p>
+        <p class="govuk-body">Areas the Scottish Government, Welsh Government, and the Northern Ireland Executive are responsible for, include:</p>
 
-            <p class="govuk-body"><%= link_to "Understand how your council works", "/understand-how-your-council-works/types-of-council/", class: "govuk-link" %></p>
-          </div>
-        </div>
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Parliament",
-              heading_level: 3,
-              margin_bottom: 5,
-            } %>
+        <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
```

Comment:
Same here with the [list](https://components.publishing.service.gov.uk/component-guide/list)